odra-casper/wasm-env/src/host_functions.rs

```diff
@@ -715,3 +710,115 @@ fn get_named_arg_size(name: &str) -> Result<usize, ApiError> {
         _ => Err(ApiError::from(ret as u32))
     }
 }
+
+fn caller_info_to_caller(info: CallerInfo) -> Caller {
```

Comment:
Make this function to return Result with possible error. Remove all `expect` calls. We should have as little panics in the code as possible.